.github/workflows/workflow.yaml

```diff
@@ -51,6 +54,48 @@ jobs:
             ./bazel-ci-bin/tests/run-tests
             bazel coverage //tests/...
             '
+
+  test-nixpkgs-cross:
+    name: Build & Test - Cross
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Mount Bazel cache
+        uses: actions/cache@v2
+        with:
+          path: ~/repo-cache
+          key: repo-cache-${{ runner.os }}-nixpkgs-${{ env.cache-version }}
+      - uses: cachix/install-nix-action@v12
+        with:
+          nix_path: nixpkgs=./nixpkgs/default.nix
+          extra_nix_config: |
+            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+            extra-substituters = https://hydra.iohk.io
+      - name: Configure
+        env:
+          BUILDBUDDY_API_KEY: ${{ secrets.BUILDBUDDY_API_KEY }}
+        run: |
+          cat >.bazelrc.local <<EOF
+          # Enabling --config=ci causes cross-compilation to crawl
```

Comment:
For reference, this is tracked in https://github.com/tweag/rules_haskell/issues/1524